docs/ide/javascript-intellisense.md

```diff
@@ -117,7 +119,7 @@ Because JavaScript and TypeScript are now based on the same language service, th
 
 Below, we show a simple example of a TypeScript definition file providing such type information (via an interface) to a JavaScript file in the same project (using a `JsDoc` tag).
 
-<img src="https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/images/decl1.png" height="400" width="640" alt="TypeScript definition file" />
+![TypeScript definition file](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/images/decl1.png)
```

Comment:
This loses the size, but it might be good to move the image to this repo and resize it instead of references another project wiki